src/detect-flow.c

```diff
@@ -296,6 +300,13 @@ static DetectFlowData *DetectFlowParse(
                 }
                 fd->flags |= DETECT_FLOW_FLAG_ONLY_FRAG;
                 fd->match_cnt++;
+            } else if (strcasecmp(args[i], "elephant") == 0) {
+                if (fd->flags & DETECT_FLOW_FLAG_ELEPHANT) {
+                    SCLogError("flow is already marked as elephant");
```

Comment:
this about keyword parsing, so I would expect something like "duplicate elephant setting"